.dockerignore

```
@@ -138,13 +138,19 @@ temp/
 test.py
 
 # Other stuff
+.dockerignore
 .env.example
 .gitignore
 .github/
 app.json
 CHANGELOG.md
+Dockerfile
+docker-compose.yml
+LICENSE
+PRIVACY.md
 Procfile
 pyproject.toml
 README.md
 Pipfile
 Pipfile.lock
+SPONSORS.json
```

```suggestion
.dockerignore
.env.example
.git/
.gitignore
.github/
app.json
CHANGELOG.md
Dockerfile
docker-compose.yml
Procfile
pyproject.toml
README.md
Pipfile
Pipfile.lock
```

Added `.git/` to the ignored list, as suggested.

Unignored `LICENSE`, `PRIVACY.md`, and `SPONSORS.json`. They do not affect the functionality of the bot, but due to their respective purposes, we prefer to include them in the image.